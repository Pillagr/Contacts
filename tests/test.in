Meier, Rohan. 02/17/12 11:15
"220 Burns St"  "Milton"	"Wyoming"	"20057"
(WORK)1-781-990-9802
rohan.meier@gmail.com
my brother|
Meier, Geoffrey. 11/01/09 10:00
"51 Avalon Rd"	"Boston"	"Mass"	"02186"
(HOME)1-617-785-9800
geoffrey.meier@gmail.com
my dad|
Meier, Sachin. 12/01/09 10:00
"50 Randolph Rd"	"Milton"	"Mass"	"02186"
(HOME)1-617-785-9800
sachin.meier@gmail.com
me|
Meier, Divya. 01/01/09 10:00
"49 Milton Rd"	"Boston"	"Mass"	"02186"
(HOME)1-617-785-9800
Divya.meier@gmail.com
my sister|
Meier, Sindu. 01/01/09 10:00
"49 Milton Rd"	"Milton"	"Mass"	"02186"
(HOME)1-617-785-9800
sindu.meier@gmail.com
my mom|