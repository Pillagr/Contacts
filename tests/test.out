Meier, Rohan. 02/17/12 11:15
"220 Burns St"	"Milton"	"Wyoming"	"20057"
(WORK)1-781-990-9802
rohan.meier@gmail.com
my brother|
Meier, Sachin. 11/01/09 10:00
"51 Avalon Rd"	"Boston"	"Mass"	"02186"
(HOME)1-617-785-9800
sachin.meier@gmail.com
x|
Meier, Sachin. 12/01/09 10:00
"51 Avalon Rd"	"Boston"	"Mass"	"02186"
(HOME)1-617-785-9800
sachin.meier@gmail.com
x|
Meier, Sachin. 01/01/09 10:00
"51 Avalon Rd"	"Boston"	"Mass"	"02186"
(HOME)1-617-785-9800
sachin.meier@gmail.com
x|